<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pinUI</class>
 <widget class="QDialog" name="pinUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(192, 234, 227);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);
text-transform: uppercase;
background-color: rgb(246, 186, 111);
border-color: rgb(145, 187, 145);
border: 2px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Syötä PIN-koodi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255,255,255);
border: 5px;
border-color: rgb(135, 203, 185);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>syötä pin-koodi tähän</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242,242,242);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>poistu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
